rapidpro_community_portal/static/css/main.css

```diff
@@ -35,6 +35,7 @@ footer{
 a{
   text-decoration:none;
   color: #069;
+  font-weight: 700;
```

Comment:
Too me this creates too much bold on the filtering options and listing for pages like https://rapidpro-staging.cakt.us/stories/mother-reminders/. Kalee specifically mentions blue text in her request, so I'm thinking the intent was only to bold places where links aren't already styled to look like buttons. I think we either need to make this font-weight addition more restrictive or adjust the css for the "button" version to not inherit this.